src/components/ApplicationViews.js

```diff
@@ -24,10 +25,14 @@ export const ApplicationViews = () => {
             margin: "5rem 2rem",
             lineHeight: "1.75rem"
         }}>
+        <PostProvider>
+          <Route path="/" component={PostList} />
```

Comment:
I think we may want this to be an `exact path` instead of just `path`. We want this list to render only when the user is at `/`, but with this as it is, the `PostList` will be rendering regardless of the path.